python/cudf_polars/cudf_polars/dsl/expressions/rolling.py

```diff
@@ -62,6 +78,15 @@ def to_request(
     return plc.rolling.RollingRequest(col.obj, min_periods, value.agg_request)
 
 
+def _by_exprs(b: Expr | tuple) -> list[Expr]:
```

Comment:
suggestion: this could be a generator since it appears you're re-iterating over the result again further down.